src/components/Navigation/navigation.js

```diff
@@ -37,17 +37,9 @@ export const HorNav = () => {
     [dispatch]
   );
 
-  const [navClass, setNavClass] = useState('nav-top');
-
-  useEffect(() => {
-    window.addEventListener('scroll', () => {
-      setNavClass(`${navClass} nav-white-bg`);
-    });
-  }, [navClass]);
-
   return (
```

Comment:
Removing this useEffect was an excellent call.  When adding events in useEffect, remember that you have to provide the cleanup function so when the component was unmounted, it would remove the event listener.